gradle-conjure/src/main/java/com/palantir/gradle/conjure/Dependencies.java

```diff
@@ -19,12 +19,15 @@
 final class Dependencies {
 
     /** Make the old Java8 @Generated annotation available even when compiling with Java9+. */
-    static final String ANNOTATION_API = "jakarta.annotation:jakarta.annotation-api:1.3.5";
+    static final String ANNOTATION_API_JAKARTA = "jakarta.annotation:jakarta.annotation-api:1.3.5";
+
+    static final String ANNOTATION_API_JAVAX = "javax.annotation:javax.annotation-api:1.3.2";
 
     static final String CONJURE_JAVA_LIB = "com.palantir.conjure.java:conjure-lib";
     static final String CONJURE_UNDERTOW_LIB = "com.palantir.conjure.java:conjure-undertow-lib";
     static final String DIALOGUE_TARGET = "com.palantir.dialogue:dialogue-target";
-    static final String JAXRS_API = "jakarta.ws.rs:jakarta.ws.rs-api";
+    static final String JAXRS_API_JAKARTA = "jakarta.ws.rs:jakarta.ws.rs-api";
+    static final String JAXRS_API_JAVAX = "javax.ws.rs:javax.ws.rs-api";
```

Comment:
Let's specify versions for these, otherwise builds may fail if no constraints exist for javax.ws.rs...etc. Otherwise the jakarta segmentation plugin must be applied first.

Reply:
fixed